Parsers/ASimAuthentication/ProductParsers/AuthenticationWindowsEvents.yaml

```diff
@@ -0,0 +1,136 @@
+                              '0xc000010b', 'STATUS_INVALID_LOGON_TYPE','Logon violates policy',
+                              '0xc000015b', 'STATUS_LOGON_TYPE_NOT_GRANTED','Logon violates policy',
+                              '0xc000018b', 'STATUS_NO_TRUST_SAM_ACCOUNT','Logon violates policy',
+                              '0xc0000224', 'STATUS_PASSWORD_MUST_CHANGE','Other',
+                              '0xc0000234', 'STATUS_ACCOUNT_LOCKED_OUT','User locked',
+                              '0xc00002ee', 'STATUS_UNFINISHED_CONTEXT_DELETED','Other'];
+                              let WinLogon=(){ // TODO: update later "Data" to "EventData", and add EventOriginalUid and AccountType
+                                  WindowsEvent
+                                  | where Provider == 'Microsoft-Windows-Security-Auditing'
+                                  | where     EventID in (LogonEvents) or EventID in (LogoffEvents)
+                                  | extend    LogonProtocol = tostring(Data.AuthenticationPackageName),
+                                              SrcDvcIpAddr = tostring(Data.IpAddress),
+                                              TargetPortNumber = toint(Data.IpPort),
+                                              LogonGuid = tostring(Data.LogonGuid),
+                                              LogonType = tostring(Data.LogonType),
+                                              ActingProcessCreationTime = Data.ProcessCreationTime,
+                                              ActingProcessId = tostring(toint(Data.ProcessId)),
+                                              ActingProcessName = tostring(Data.ProcessName),
+                                              Status = tostring(Data.Status),
+                                              ActorUserDomainName = Data.SubjectDomainName,
+                                              ActorSessionId = tostring(Data.SubjectLogonId),
+                                              ActorUsername = tostring(Data.SubjectUserName),
+                                              ActorUserId = tostring(Data.SubjectUserSid),
+                                              SubStatus = tostring(Data.SubStatus),
+                                              TargetDomainName = tostring(Data.TargetDomainName),
+                                              TargetSessionId = tostring(Data.TargetLogonId),
+                                              TargetUserId = tostring(Data.TargetUserSid),
+                                              TargetUsername = tostring(Data.TargetUserName),
+                                              SrcDvcHostname = tostring(Data.WorkstationName),
+                                              ActorUserAccount = tostring(Data.SubjectAccount),
+                                              TargetUserAccount = tostring(Data.TargetAccount),
+                                              EventProduct = "Microsoft-Windows-Security-Auditing"
```

Comment:
Should be "Security Events". It is the product as other sources delivering this event. If we want to change, we need to change SecurityEvent as well.